cns/grpc/proto/server.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package cns;
+
+option go_package = "cns/grpc/cnsv1alpha";
+
+// The Container Network Service (CNS) exposes a set of operations that allow the Delegated Network Controller (DNC) to manage
+// and monitor nodes in an orchestrator's infrastructure.
+
+// CNSService defines the gRPC service exposed by CNS to interact with DNC.
+service CNSService {
```

Comment:
nit: instead of using the word service twice, can we rename this service to `CNS`? The type of the object already tells us that its a service
```suggestion
service CNS {
```

Reply:
+1, also the S in CNS stands for Service already 🙂

Reply:
Got it, made the changes thanks!